TwilioVerifyDemo/TwilioVerifyDemo/TwilioVerify/TwilioVerifyAdapter.swift

```diff
@@ -27,7 +27,9 @@ class TwilioVerifyAdapter {
     #if DEBUG
       builder = builder.enableDefaultLoggingService(withLevel: .all)
     #endif
-    twilioVerify = try builder.build()
+    twilioVerify = try builder
+      .setAccessGroup("group.company.identifier")
```

Comment:
We could add a TODO for setting the customer's correct access group. Should we use the access group for the sample app in order to generate builds for testing?

Reply:
And we can include also where to update the access group for the extension

Reply:
Added #warning instead, that would be more visible. Also mentioned the NotificationExtension.